src/sentry/interfaces/spans.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import absolute_import
+
+__all__ = ('Spans', 'Span')
+
+from sentry.interfaces.base import Interface
+
+SPAN_KEYS = (
+    'trace_id',
+    'parent_span_id',
+    'span_id',
+    'start_timestamp',
+    'same_process_as_parent',
+    'description',
+    'tags',
+    'timestamp',
+    'op',
+    'data'
+)
+
+
+class Span(Interface):
+    """
+    Holds timing spans related to APM and tracing.
+
+    >>> {
+    >>>   'trace_id': 'a0fa8803753e40fd8124b21eeb2986b5',
+    >>>   'parent_span_id': '9c2a6db8c79068a2',
+    >>>   'span_id': '8c931f4740435fb8',
+    >>>   'start_timestamp': '2019-06-14T14:01:41Z',
+    >>>   'same_process_as_parent': true,
+    >>>   'description': 'http://httpbin.org/base64/aGVsbG8gd29ybGQK GET',
+    >>>   'tags': { 'http.status_code': 200, 'error': false },
+    >>>   'timestamp': '2019-06-14T14:01:41Z',
+    >>>   'op': 'http',
+    >>>   'data': {
+    >>>     'url': 'http://httpbin.org/base64/aGVsbG8gd29ybGQK',
+    >>>     'status_code': 200,
+    >>>     'reason': 'OK',
+    >>>     'method': 'GET'
+    >>>   }
+    >>> }
+    """
+    @classmethod
+    def to_python(cls, data):
+        for key in SPAN_KEYS:
+            data.setdefault(key, None)
+        return cls(**data)
+
+
+class Spans(Interface):
+    """
+    Contains a list of Span interfaces
+    """
+    display_score = 1950
```

Comment:
What are these used for?

Reply:
@wedamija looks like for sorting the interfaces: https://github.com/getsentry/sentry/blob/a4df91901320fcaa78a5ec226dbe931436c808c3/src/sentry/api/serializers/models/event.py#L75-L76

Reply:
Interfaces with higher display order are returned first in the `event.entries` list, which in turn changes the order interface UI widgets are displayed.